assets/js/base/context/cart-checkout/checkout-state/index.js

```diff
@@ -137,15 +137,15 @@ export const CheckoutStateProvider = ( {
 
 	// emit events.
 	useEffect( () => {
-		const { hasError, status } = checkoutState;
+		const { status } = checkoutState;
 		if ( status === STATUS.PROCESSING ) {
 			removeNotices( 'error' );
 			emitEvent(
 				currentObservers.current,
 				EMIT_TYPES.CHECKOUT_PROCESSING,
 				{}
 			).then( ( response ) => {
-				if ( response !== true || hasError ) {
+				if ( response !== true ) {
```

Comment:
hasError doesn't matter in this context because if there's an error from somewhere else, it's already handled elsewhere. Here we only care about any errors passed through on the event emitter.